UI/forms/OBSPermissions.ui

```diff
@@ -274,7 +274,7 @@
               <number>25</number>
              </property>
              <item>
-              <widget class="QWidget"/>
+              <widget class="QWidget" name="widget" native="true"/>
```

Comment:
Why even are there blank nameless widgets here? Layout reasons? @gxalpha ?

I feel like there's a mechanism for "add an element to the second column in a layout" without having to spawn an empty widget. Or else a spacer item should work instead of a full widget? If so, that change would be out of scope for this PR, but we could change it in a separate PR.

If not, then yeah, they'll need unique names, which again, we could probably fix in a separate PR to keep the scope/intent of this PR constrained.

Reply:
Would need to dig in to see what was going on there. I mostly took the layout from an earlier attempt at this dialog (by Warchamp and Matt) and then modified it, it’s quite possible that I copied this part and made bad modifications here.

Reply:
See #10473

Reply:
Should be resolved now.